README.md

```diff
@@ -1,58 +1,48 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+"OpenTouryo", is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.
```

Comment:
@daisukenishino 

Please revise as follows:

*OpenTouryo* is an application framework for .NET Framework which was developed using the accumulated know-how with a longstanding application development results.